src/tests/ftest/pool/management_race.py

```diff
@@ -103,11 +100,10 @@ def test_pool_management_race(self):
         self.pool = []
         for pool_number in range(num_pools):
             label = pool_label_prefix + str(pool_number)
-            pool = self.get_pool()
-            pool.set_property(prop_name="label", prop_value=label)
-            pool.label.update(label)
-            self.pool.append(pool)
-            self.log.info("==(1.%d) pool created, %d.", pool_number, pool)
+            self.pool.append(self.get_pool(create=False))
+            self.pool[-1].label.update(label)
+            self.pool[-1].create()
+            self.log.info("==(1.%d) pool created, %d.", pool_number, self.pool[-1])
```

Comment:
```suggestion
        self.add_pool_qty(quantity=num_pools)
```
You can replace Line 99 to 106 with this one line.